src/app/components/MediaLoader/utils/buildSetting.test.ts

```diff
@@ -18,287 +20,366 @@ describe('buildSettings', () => {
+        ...baseSettings,
+        blocks: aresMediaBlocks as MediaBlock[],
+      });
 
-    jest.spyOn(window, 'window', 'get').mockImplementation(() => mockWindowObj);
-
-    const result = buildSettings({
-      ...baseSettings,
-      blocks: aresMediaBlocks as MediaBlock[],
-    });
-
-    expect(result?.playerConfig.mediator).toBe(undefined);
-  });
-
-  it('Should NOT include the mediator parameter if we are on a live url.', () => {
-    const mockWindowObj = {
-      location: {
-        hostname: 'https://www.bbc.com/',
-      },
-    } as Window & typeof globalThis;
-
-    jest.spyOn(window, 'window', 'get').mockImplementation(() => mockWindowObj);
-
-    const result = buildSettings({
-      ...baseSettings,
-      blocks: aresMediaBlocks as MediaBlock[],
-    });
-
-    expect(result?.playerConfig.mediator).toBe(undefined);
-  });
-
-  it('Should return null if the AresMedia block contains invalid data.', () => {
-    const sampleBlock = [
-      {
-        model: { blocks: [{ model: { versions: [] } }] },
-      },
-    ];
-    const result = buildSettings({
-      ...baseSettings,
-      // @ts-expect-error - we are testing an invalid block
-      blocks: sampleBlock,
-    });
-
-    expect(result).toBe(null);
-  });
-
-  it('Should return the statsObject with required values for tracking.', () => {
-    const result = buildSettings({
-      ...baseSettings,
-      blocks: aresMediaBlocks as MediaBlock[],
+      expect(result?.playerConfig.statsObject).toStrictEqual({
+        clipPID: 'p01k6msp',
+        destination: 'WS_NEWS_LANGUAGES',
+        producer: 'MUNDO',
+      });
     });
 
-    expect(result?.playerConfig.statsObject).toStrictEqual({
-      clipPID: 'p01k6msp',
-      destination: 'WS_NEWS_LANGUAGES',
-      producer: 'MUNDO',
+    it('Should return required settings for tracking.', () => {
+      const result = buildSettings({
+        ...baseSettings,
+        blocks: aresMediaBlocks as MediaBlock[],
+      });
+
+      expect(result?.playerConfig).toHaveProperty(
+        'counterName',
+        'live_coverage.testID.page',
+      );
+      expect(result?.playerConfig).toHaveProperty('appName', 'news-mundo');
+      expect(result?.playerConfig).toHaveProperty('appType', 'responsive');
+      expect(result?.playerConfig.playlistObject).toHaveProperty(
+        'title',
+        'Five things ants can teach us about management',
+      );
     });
   });
 
-  it('Should return required settings for tracking.', () => {
-    const result = buildSettings({
-      ...baseSettings,
-      blocks: aresMediaBlocks as MediaBlock[],
+  describe('OnDemandTv', () => {
+    it('Should process a On Demand TV block into a valid playlist item.', () => {
+      const mockWindowObj = {
+        location: {
+          hostname: 'https://www.bbc.com/',
+        },
+      } as Window & typeof globalThis;
+
+      jest
+        .spyOn(window, 'window', 'get')
+        .mockImplementation(() => mockWindowObj);
```

Comment:
Not sure if we need this, let's try to comment out / delete, and hope that the test still passes! 🤞